virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -13,9 +13,9 @@ Review this section before you install {VirtProductName} to ensure that your clu
 You can use any installation method, including user-provisioned, installer-provisioned, or assisted installer, to deploy {product-title}. However, the installation method and the cluster topology might affect {VirtProductName} functionality, such as snapshots or live migration.
 ====
 
-.Single Node OpenShift behavior
+.{sno-first} behavior
 
-You can install {VirtProductName} on a single node cluster, also known as xref:../../installing/installing_sno/install-sno-preparing-to-install-sno.adoc#install-sno-about-installing-on-a-single-node_install-sno-preparing[Single Node OpenShift] (SNO). SNO does not support high availability, which results in the following differences:
+You can install {VirtProductName} on a single-node cluster, see xref:../../installing/installing_sno/install-sno-preparing-to-install-sno.adoc#install-sno-about-installing-on-a-single-node_install-sno-preparing[About {sno}] for more information. {sno-first} does not support high availability, which results in the following differences:
```

Comment:
You could split the first two sentences in this line into two sentences, to facilitate localisation. For example:

> You can install {VirtProductName} on a single-node cluster. See xref:../../installing/installing_sno/install-sno-preparing-to-install-sno.adoc#install-sno-about-installing-on-a-single-node_install-sno-preparing[About {sno}] for more information.